pep-0001.txt

```diff
@@ -123,10 +123,10 @@ Start with an idea for Python
 
 The PEP process begins with a new idea for Python.  It is highly
 recommended that a single PEP contain a single key proposal or new
-idea. Small enhancements or patches often don't need
-a PEP and can be injected into the Python development workflow with a
-patch submission to the Python `issue tracker`_. The more focused the
-PEP, the more successful it tends to be.  The PEP editors reserve the
+idea; the more focused the PEP, the more successful it tends to be.
+Small enhancements and bug fixes often don't need a PEP and
```

Comment:
@AA-Turner suggested changing "small" to "most"

Reply:
```suggestion
Most enhancements and bug fixes often don't need a PEP and
```

Reply:
Sorry, I made that change but it got lost when I patched my version on top of the further changes on this PR.